.gitignore

```diff
@@ -109,3 +109,6 @@ doc/source/quickstart/.ipynb_checkpoints/
 dist
 .python-version
 answer_nosetests.xml
+yt/utilities/lib/cosmology_time.cpp
```

Comment:
You should avoid editting .gitignore unless absolutely necessary. This will push to the main yt repo when we try to merge in our changes.